kbi/0006/index.rst

```diff
@@ -0,0 +1,70 @@
+.. index::
+   single: special remote; reconfigure
+   single: git-annex; git special remote
```

Comment:
I believe this would be a good use case for `triple:`

```suggestion
.. index::
   triple: git-annex; special remote; reconfigure
```

![image](https://user-images.githubusercontent.com/136479/232712252-29f3eae6-d6da-40d3-bbb6-e7e60b17e1da.png)

Reply:
mhh, the image link points into the void, but if you think it makes sense, go for it :)

Reply:
I creates all pairs from the triple.